bdk-ffi-bindgen/Cargo.toml

```
@@ -6,5 +6,5 @@ edition = "2021"
 [dependencies]
 anyhow = "=1.0.45" # remove after upgrading to next version of uniffi
 structopt = "0.3"
-uniffi_bindgen = "0.19.5"
+uniffi_bindgen = "=0.20.0"
```

This one also doesn't need to be pinned:
```suggestion
uniffi_bindgen = "0.20.0"
```